From 376ee72c8d97486dd195df2f92685894d6d4e1a1 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Wed, 21 Aug 2019 17:00:27 +0900
Subject: [PATCH] docs: add captured port types for spp_pcap

Spp_pcap only supports two port types, phy and ring, currently. However,
There is no explanation for the supported types. This update is to add
descriptions for.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 docs/guides/design/spp_secondary.rst | 8 ++++----
 docs/guides/gsg/howto_use.rst        | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/docs/guides/design/spp_secondary.rst b/docs/guides/design/spp_secondary.rst
index b89d8b4..e09c130 100644
--- a/docs/guides/design/spp_secondary.rst
+++ b/docs/guides/design/spp_secondary.rst
@@ -209,11 +209,11 @@ specific port. It is aimed to capture up to 10Gbps packets.
 ``spp_pcap`` in which ``spp_pcap`` process receives packets from ``phy:0``
 for capturing.
 
-``spp_pcap`` provides packet capturing capability as a SPP secondary process.
-``spp_pcap`` has one manager thread like spp_vf, and has two types of worker
-threads unlike spp_vf.
-
+.. note::
 
+   ``spp_pcap`` supports only two types of ports for capturing, ``phy``
+   and ``ring``, currently.
+    
 .. _figure_spp_pcap_overview:
 
 .. figure:: ../images/design/spp_pcap_overview.*
diff --git a/docs/guides/gsg/howto_use.rst b/docs/guides/gsg/howto_use.rst
index dee0a84..5e461eb 100644
--- a/docs/guides/gsg/howto_use.rst
+++ b/docs/guides/gsg/howto_use.rst
@@ -540,7 +540,7 @@ options of ``spp_pcap``.
 
 * ``--client-id``: Client ID unique among secondary processes.
 * ``-s``: IPv4 address and secondary port of spp-ctl.
-* ``-c``: Captured port, e.g. ``phy:0``, ``ring:1`` or so.
+* ``-c``: Captured port. Only ``phy`` and ``ring`` are supported.
 * ``--out-dir``: Optional. Path of dir for captured file. Default is ``/tmp``.
 * ``--fsize``: Optional. Maximum size of a capture file. Default is ``1GiB``.
 
-- 
2.17.1

